src/vivarium/framework/results/context.py

```diff
@@ -61,7 +61,7 @@ def add_stratification(
         name: str,
         sources: List[str],
         categories: List[str],
-        excluded_categories: Optional[List[str]],
+        excluded_categories: List[str],
```

Comment:
Why did you change this from None to an empty list?

Reply:
It just seemed more consistent w/ the other list arguments defauling to [] instead of None.

Reply:
Put it back to None b/c it conveys different information than an empty list.